llvm/tools/llvm-exegesis/lib/BenchmarkRunner.cpp

```diff
@@ -384,6 +393,31 @@ class SubProcessFunctionExecutorImpl
     return make_error<SnippetSignal>(ChildSignalInfo.si_signo);
   }
 
+  void setCPUAffinityIfRequested() const {
```

Comment:
[nit] `static void setCPUAffinity(int) {` here, and `if (BenchmarkProcessCPU.has_value()) { setCPUAffinity(*value); }` at the call site.